mod-mosaic/schemas/mosaic_order_request.json

```diff
@@ -0,0 +1,150 @@
+    },
+    "title": {
+      "description": "title of the material",
+      "type": "string"
+    },
+    "author": {
+      "description": "the name of a author to the material",
+      "type": "string"
+    },
+    "publicationDate": {
+      "description": "date (year) of the material's publication",
+      "type": "string"
+    },
+    "edition": {
+      "description": "edition of the material",
+      "type": "string"
+    },
+    "productId": {
+      "description": "The actual product identifier",
+      "type": "string"
+    },
+    "receivingNote": {
+      "description": "notes regarding receiving instructions",
+      "type": "string"
+    },
+    "userLimit": {
+      "description": "the concurrent user-limit",
+      "type": "string"
+    },
+    "requesterName": {
+      "description": "who requested this purchase order line",
+      "type": "string"
+    },
+    "selectorName": {
+      "description": "who selected this material",
+      "type": "string"
+    },
+    "poLineNote": {
+      "description": "note for poLIne",
+      "type": "string"
+    },
+    "poLineDescriptor": {
+      "description": "descriptor for poLine",
+      "type": "string"
+    },
+    "renewalNote": {
+      "description": "Renewal note for this purchase order line",
+      "type": "string"
+    },
+    "referenceNumbers": {
+      "description": "Collection of reference number items",
+      "$ref": "../../common/schemas/reference_numbers.json"
+    },
+    "listUnitPrice": {
+      "description": "The per-item list price for physical or resources of 'Other' order format",
+      "type": "number"
+    },
+    "listUnitPriceElectronic": {
+      "description": "The e-resource per-item list price",
+      "type": "number"
+    },
+    "currency": {
+      "description": "An ISO currency code",
+      "type": "string"
+    },
+    "quantityPhysical": {
+      "description": "Quantity of physical items or resources of 'Other' order format in this purchase order line",
+      "type": "integer"
+    },
+    "quantityElectronic": {
+      "description": "Quantity of electronic items in this purchase order line",
+      "type": "integer"
+    },
+    "folioOrderTemplateId": {
+      "description": "UUID of the order template",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "locationId": {
+      "description": "location of the material",
+      "$ref" : "../../common/schemas/uuid.json"
+    },
+    "fundId": {
+      "description": "fund of poLine",
+      "$ref" : "../../common/schemas/uuid.json"
+    },
+    "expenseClassId": {
+      "description": "expense class of the material",
+      "$ref" : "../../common/schemas/uuid.json"
+    },
+    "materialTypeId": {
+      "description": "material type of the material",
+      "$ref" : "../../common/schemas/uuid.json"
+    },
+    "acquisitionUnitId": {
+      "description": "acquisition unit",
+      "$ref" : "../../common/schemas/uuid.json"
+    },
+    "billTo": {
```

Comment:
Even if our existing properties name like "billTo" instead of "billToId" I think it is makes sense to remain them as we have already in CompositePurchaseOrder and PoLine schemas, it will simplify mapping, so no changes are needed here